snmp/datadog_checks/snmp/data/default_profiles/netgear-readynas.yaml

```
@@ -0,0 +1,131 @@
+extends:
+  - _generic-ucd.yaml
+  - netgear.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.4526.100.12.*
+  - 1.3.6.1.4.1.4526.100.15.*
+  - 1.3.6.1.4.1.4526.100.16.*
+  - 1.3.6.1.4.1.4526.22.*
+metrics:
+  # - MIB: READYNASOS-MIB
+  #   symbol:
+  #     OID: 1.3.6.1.4.1.4526.22.1.0
+  #     name: netgear.nasMgrSoftwareVersion
+  #     tag: netgear_nas_mgr_software_version # scalar string metric is not supported yet
+
+  - MIB: READYNASOS-MIB
+    table:
+      OID: 1.3.6.1.4.1.4526.22.3
+      name: diskTable
+    symbols:
+      - OID: 1.3.6.1.4.1.4526.22.3.1.6
+        name: netgear.ataError
```

```suggestion
        name: netgear.readynasos.ataError
```

I think in this case it's likely safer to prefix with both netgear + readynasos since the symbols are very generic.